docsk8s/quick-start/ls-k8s-quick-start.asciidoc

```diff
@@ -0,0 +1,267 @@
+=== Generate certificate files
+
+To enable secure communication throughout your Kubernetes resources, run the sample script to generate the CA certificate files. Details about these files are in <<sample-configuration-files,Sample configuration files>>.
+
+[source,sh]
+--
+./cert/generate_cert.sh
+--
+
+[source,sh]
+--
+Generating RSA private key, 2048 bit long modulus
+.......................+++
+...........................................................................+++
+e is 65537 (0x10001)
+Generating RSA private key, 2048 bit long modulus
+..............................................+++
+.............................................+++
+e is 65537 (0x10001)
+Signature ok
+subject=/C=EU/ST=NA/O=Elastic/CN=ServerHostName
+Getting CA Private Key
+Generating RSA private key, 2048 bit long modulus
+............+++
+.......................................................................................................................................+++
+e is 65537 (0x10001)
+Signature ok
+subject=/C=EU/ST=NA/O=Elastic/CN=ClientName
+Getting CA Private Key
+--
+
+Your `logstash-k8s-gs/cert` folder should now contain a set of certificate files, including `client` certificates for Filebeat and Metricbeat, and `server` certificates for Logstash. 
+
+The parent `logstash-k8s-gs` directory also has a new `001-secret.yaml` resources file that stores a hash of the client and server certificates.
+
+image::./images/gs-cert-files.png[generated CA certificate files]
+
+[float]
+[[qs-create-elastic-stack]]
+=== Create an Elastic Stack
+
+Now that your environment and certificates are set up, it's time to create an Elastic Stack. Run the following command to deploy the example using the sample CRDs:
+
+[source,sh]
+--
+kubectl apply -f .
+--
+
+The resources are created:
+
+[source,sh]
+--
+elasticsearch.elasticsearch.k8s.elastic.co/demo created
+configmap/logstash-pipeline created
+configmap/logstash-config created
+secret/logstash-beats-tls created
+deployment.apps/logstash created
+service/logstash created
+horizontalpodautoscaler.autoscaling/logstash created
+beat.beat.k8s.elastic.co/demo created
+beat.beat.k8s.elastic.co/demo configured
+kibana.kibana.k8s.elastic.co/demo created
+--
+
+[source,sh]
+--
+kubectl get pods
+--
+
+The pods are starting up. You may need to wait a minute or two for all of them to be ready.
+
+[source,sh]
+--
+NAME                                    READY   STATUS    RESTARTS       AGE
+demo-beat-filebeat-7f4d97f69f-qkkbl     1/1     Running   0              42s
+demo-beat-metricbeat-59f4b68cc7-9zrrn   1/1     Running   0              39s
+demo-es-default-0                       1/1     Running   0              41s
+demo-kb-d7f585494-vbf6s                 1/1     Running   0              39s
+elastic-operator-0                      1/1     Running   4 (164m ago)   13d
+logstash-7974b9ccb9-jd5xl               1/1     Running   0              42s
+--
+
+[float]
+[[qs-view-monitoring-data]]
+=== View the monitoring data
```

Comment:
This feels a little ambiguous - does the monitoring data mean stack monitoring? Or the data we are putting in the `kube-apiserver`? 

Maybe we can split this into "verify data" and "view stack monitoring"? Or something along those lines

Reply:
Since I've removed the part about viewing indices I think now we're only viewing stack monitoring info (i.e., what appears in the Kibana visualization: JVM Heap, CPU, and System Load data), so I've updated the text to `View the stack monitoring data`.